src/encoded/schemas/file.json

```diff
@@ -271,6 +271,15 @@
                 }
             }
         },
+        "cropped_read_length_tolerance": {
+            "comment": "Only bam files with cropped_read_length specified can have cropped_read_length_tolerance.",
```

Comment:
does "bam" supposed to be capitalized ("BAM")?

Reply:
I don't think it has to be.